packit/cli/init.py

```diff
@@ -31,20 +41,49 @@
     is_flag=True,
     help="Reset config to default if already exists.",
 )
+@click.option(
+    "--with_precommit",
```

Comment:
```suggestion
    "--force-precommit",
```

Given the help message I'd go with something like this.